### PR TITLE
YouTube Subproject Genesis

### PR DESCRIPTION
**Pull request by chris-short**
The Stream Team

Includes fixes to links to youtube-guideline.md that would break as a result of this commit.

~~Never created a subproject before so flog me as necessary.~~

Second attempt at a subproject so flog me again.

Signed-off-by: Chris Short <chris@chrisshort.net>

### CODE REVIEW

File: communication/youtube/youtube-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+Kubernetes [Community meeting], [Office Hours], [Meet our Contributors] and
+others.
+
+To better serve the community, [collaboration] has been enabled to share the
+management of the playlists. Anyone with the appropriate link to the particular
+playlist can upload videos _to that particular playlist_ (links & playlists are
+one-to-one).
+
+Each group's playlist link will be shared with the group's leadership via Slack
+and the group leadership Google Group. Other playlists links, for example Office Hours, will be shared with the appropriate point(s) of contact.
```
[the remark this answers is not in the document]
Comment (chris-short) [reply communication/youtube/youtube-guidelines.md]: @mrbobbytables Can you ping me on Slack this morning to help me get this PR fixed up proper?

[the remark this answers is not in the document]
Comment (mrbobbytables) [reply communication/youtube/youtube-guidelines.md]: @chris-short I can't this morning, going to be away from computer most of the day. I should be able to Monday/Tuesday though.

**Comment by chris-short** [comment]
https://github.com/kubernetes/community/pull/3702

**Pull request closed without merging**